### PR TITLE
openvdb: update to 8.0.1

### PR DESCRIPTION
**Pull request by jasonliu--**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010 x86_64
Xcode 8.2.1 8C1002

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: graphics/openvdb/Portfile [hunk 1]
```diff
@@ -78,13 +77,6 @@ variant tests description {Build unit tests} {
 
 variant docs description {Build documentation} {
     depends_build-append    port:doxygen
-    post-patch {
-        reinplace "s|doc/html|share/doc/${name}/html|" \
-            ${worksrcpath}/doc/CMakeLists.txt
-        reinplace -E \
-            "s|(DESTINATION )doc( MESSAGE_NEVER)|\\1share/doc/${name}\\2|" \
-            ${worksrcpath}/doc/CMakeLists.txt
-    }
```
Comment (jasonliu--) [inline graphics/openvdb/Portfile:87]: OpenVDB 8.0.1 no longer hard codes the doc paths and instead uses `${CMAKE_INSTALL_DOCDIR}`, so these replacements are no longer needed.

File: graphics/openvdb/Portfile [hunk 2]
```diff
@@ -7,8 +7,7 @@ PortGroup               active_variants 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               boost 1.0
 
-github.setup            AcademySoftwareFoundation openvdb 8.0.0 v
-revision                1
```
Comment (reneeotten) [inline graphics/openvdb/Portfile:11]: please leave the `revision` field but reset it to `0`

Comment (jasonliu--) [reply graphics/openvdb/Portfile:11]: Done

**Comment by jasonliu--** [comment]
Note that [Marcus is currently unavailable for PR reviews](https://github.com/macports/macports-ports/pull/13273#issuecomment-990059380), so this PR can be merged whenever any committers are ready.

**Review by reneeotten — APPROVED** [review]

**Pull request merged**